.gitignore

```diff
@@ -0,0 +1,9 @@
+venv/
```

Comment:
When I saw it, I realized that we need to have template also for `.gitignore`.  [MP](https://code.launchpad.net/~rgildein/bootstack-charms-spec/+git/bootstack-charms-spec/+merge/434795)

This not blocking this PR to be merged.